models/staging/src_amazon_selling_partner.yml

```diff
@@ -0,0 +1,532 @@
+version: 2
+
+sources:
+  - name: amazon_selling_partner
+    database: '{% if target.type not in ("spark") %}{{ var("amazon_selling_partner_database", target.database) }}{% endif %}'
+    schema: '{{ var("amazon_selling_partner_schema", "amazon_selling_partner") }}'
+    tables:
+      - name: orders
```

Comment:
Won't these tables that can be turned on/off need the enabled config?

Reply:
This applies to all other tables in this yml that can be turned on/off

Reply:
ah yep updated